sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -385,3 +386,16 @@ def test_make_moons():
         dist_sqr = ((x - center) ** 2).sum()
         assert_almost_equal(dist_sqr, 1.0,
                             err_msg="Point is not on expected unit circle")
+
+
+def test_make_circles():
+    f = 0.3
+    X, y = make_circles(7, shuffle=False, noise=None, factor=f)
```

Comment:
Thank you for the PR! 

It might be useful to test with both `n_samples` odd and even, to make sure this change doesn't break the previous behaviour.

Reply:
Thanks for the advice!
I'll also add an additional test then, which tests whether the samples were correctly distributed across both classes.

Reply:
Apart from not testing the even case, I think the current test looks fine...?